app/presenters/service_presenters.py

```diff
@@ -93,14 +96,17 @@ def get_service_id(self, service_data):
             return re.findall("....", str(id))
 
     def get_external_framework_url(self, service_data):
-        external_url = 'http://ccs-agreements.cabinetoffice.gov.uk/contracts/'
-        if service_data['frameworkName'] == 'G-Cloud 7':
-            framework_url = external_url + 'rm1557vii'
-        elif service_data['frameworkName'] == 'G-Cloud 6':
-            framework_url = external_url + 'rm1557vi'
-        else:
-            framework_url = None
-        return framework_url
+        try:
+            content_loader.load_messages(service_data['frameworkSlug'], ['urls'])
+            framework_url = content_loader.get_message(
+                service_data['frameworkSlug'],
+                'urls',
+                'framework_url'
+            )
+            return framework_url if framework_url else None
```

Comment:
This can just be done as:

```
return content_loader.get_message(
    service_data['frameworkSlug'],
    'urls',
    'framework_url'
) or None
```

but the difference is mostly stylistic.